src/main/resources/application-openshift-metering-worker.yaml

```diff
@@ -0,0 +1,16 @@
+rhsm-subscriptions:
+  # Use Spring Resource notation for this (e.g. "classpath:" or "file:")
+  account-list-resource-location: ${ACCOUNT_LIST_RESOURCE_LOCATION:}
+  account-batch-size: ${ACCOUNT_BATCH_SIZE:1}
+
+  metering:
+    prometheus:
+      metric:
+        openshift:
+          metricPromQL: >-
+            max(cluster:usage:workload:capacity_physical_cpu_cores:max:5m) without(environment, region)*on(_id)
+            group_right(prometheus) group(subscription_labels{ebs_account='%s', support=~'Premium|Standard|Self-Support|None'})
+            by (_id,ebs_account, support, usage)
+      client:
+        token: ${PROM_AUTH_TOKEN:}
+        url: ${PROM_URL:https://localhost/api/v1}
```

Comment:
Can you please add env vars for backoff?

Reply:
Yep, will do. Totally forgot about the env var.

Reply:
thanks! I could see folks wanting 1 minute max backoff, like if you have to sit around and watch if it works after making a fix. I could also see 30 min though to not spam the API :) the env var lets us change it as needed.

Reply:
DONE.